public/static/locales/fr/form-builder.json

```diff
@@ -539,7 +539,8 @@
     "missingName": {
       "message1": "Veuillez",
       "message2": "donner un nom à votre formulaire",
-      "message3": "avant de partager."
+      "message3": "avant de partager.",
+      "message4": "Activate to move focus to the form title field[FR]"
```

Comment:
@anikbrazeau A new string for translation. Feel free to replace the content with anything sounding better. The context:  the use is creating a new form, keyed the Share menu drop down and the menu dropdown currently has focus

Reply:
Is it to the title of the form (En + Fr separate) or the name of the form file (unilingual)?

Reply:
Oh right, good point. It should be the name of the form file. So "... the form name"